### PR TITLE
Add analytics to track undo/redo and other missing editor actions in Aztec

### PR DESCRIPTION
**Pull request by daniloercoli**
This is part of the changes required to fix https://github.com/wordpress-mobile/WordPress-Android/issues/10360 - It adds the following analytics to Aztec there were missing or not correctly bumped into Tracks:

- Bold
- Italics
- Align (Left | Center | Right)
- Horizontal Rule
- Undo
- Redo

To test:
- Open Aztec
- Press the buttons mentioned above and check in LogCat that analytics are bumped
- Open the option menu, and try undo/redo. Make sure that everything is bumped correctly in analytics

Please, before merging this PR, try to merge this other PR first: https://github.com/wordpress-mobile/WordPress-Android/pull/10367 🙇 

<del>Note: There is a gutenberg-mobile PR on the way, that keeps Aztec version in par: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1289</del>
Aztec PR, already merged: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1289

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java [hunk 1]
```diff
@@ -674,9 +690,11 @@ public void onToolbarFormatButtonClicked(ITextFormat format, boolean isKeyboardS
             mEditorFragmentListener.onTrackableEvent(TrackableEvent.LIST_ORDERED_BUTTON_TAPPED);
         } else if (format.equals(AztecTextFormat.FORMAT_UNORDERED_LIST)) {
             mEditorFragmentListener.onTrackableEvent(TrackableEvent.LIST_UNORDERED_BUTTON_TAPPED);
-        } else if (format.equals(AztecTextFormat.FORMAT_BOLD)) {
+        } else if (format.equals(AztecTextFormat.FORMAT_BOLD)
+                   || format.equals(AztecTextFormat.FORMAT_STRONG)) {
             mEditorFragmentListener.onTrackableEvent(TrackableEvent.BOLD_BUTTON_TAPPED);
-        } else if (format.equals(AztecTextFormat.FORMAT_ITALIC)) {
+        } else if (format.equals(AztecTextFormat.FORMAT_ITALIC)
```
Comment (maxme) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: 👍 I noticed that one and the bold/strong one when testing the other PR.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java [hunk 2]
```diff
@@ -690,6 +708,16 @@ public void onToolbarFormatButtonClicked(ITextFormat format, boolean isKeyboardS
...
+        } else if (format.equals(AztecTextFormat.FORMAT_HORIZONTAL_RULE)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.HORIZONTAL_RULE_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_LEFT)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_LEFT_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_CENTER)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_CENTER_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_RIGHT)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_RIGHT_BUTTON_TAPPED);
+        } else {
+            AppLog.w(T.EDITOR, "Aztec ToolbarFormatButtonClick not tracked in Analytics: " + format.toString());
```
Comment (maxme) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: Should we log this to Sentry to make it more visible to us?

Comment (daniloercoli) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java]: I guess it makes sense to add it, but Fragment is part of the Editor library, and doesn't have direct access to `CrashLoggingUtils`.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java [hunk 3]
```diff
@@ -529,6 +529,14 @@ public void onUndoEnabled() {
                                               getResources().getInteger(android.R.integer.config_mediumAnimTime));
     }
 
+    @Override
+    public void onUndo() {
+    }
```
Comment (marecar3) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Hey @daniloercoli, nice work here.
Maybe we can put some comment on those methods which explains why they don't have an implementation. WDYT ?

Comment (daniloercoli) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Yes, good suggestion. Addressed in ecc84afe6978e94499c88cbbcd595822d5d8ed99

Comment (marecar3) [reply libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java]: Thanks!

**Comment by marecar3** [comment]
@daniloercoli Probably we can update description for this PR as this PR: wordpress-mobile/gutenberg-mobile#1289 is closed, WDYT?

**Pull request merged**